app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -224,6 +224,7 @@ paymentHistoryOKText=OK
 paymentHistoryOverdueFooterText=Your contribution is overdue.
 paymentHistoryTitle=Your Payment History
 paymentHistoryTitle=Your Payment History
+paymentInProgress=Currently in process
```

Comment:
I believe you mean _Currently in progress_, or _Currently processing_.

Reply:
fixed